pl_bolts/utils/arguments.py

```diff
@@ -32,7 +32,7 @@ class LightningArgumentParser(ArgumentParser):
         # args.data -> data args
         # args.model -> model args
     """
-    def __init__(self, *args, ignore_required_init_args=True, **kwargs):
+    def __init__(self, *args: Any, ignore_required_init_args: bool = True, **kwargs: Any):
         """
         Args:
             ignore_required_init_args (bool, optional): Whether to include positional args when adding
```

Comment:
As we have function annotations here, the types of args should be automatically reflected in the docs, so can you remove these from docstrings?
```suggestion
            ignore_required_init_args: Whether to include positional args when adding
```

Reply:
Sure, can do this as well.